service/awsconfig/v5/resource/loadbalancer/resource.go

```diff
@@ -63,3 +55,16 @@ func (r *Resource) Name() string {
 func (r *Resource) Underlying() framework.Resource {
 	return r
 }
+
+func toLoadBalancerState(v interface{}) (LoadBalancerState, error) {
```

Comment:
You said you wanted to change the state to a pointer. We should then dispatch the same state everywhere which means we should deal with a pointer everywhere which means this here needs to be a pointer as well. The consistency of dispatching the very same type within a resource makes the implementation and understanding of it a lot easier.

Reply:
Ack. Fixed.